src/data/showcases.js

```diff
@@ -665,6 +665,15 @@ const Showcases = [
     source: null,
     tags: ["nft"],
   },
+  {
+  title: "CNFT.IO - NFT Marketplace on Cardano",
+    description:
+      "Mint, Launch, Buy and Sell Cardano NFTS on CNFT.IO. The first automated marketplace on Cardano.",
```

Comment:
Hi @itsmestale 
Please read the [About description section](https://github.com/cardano-foundation/developer-portal/blob/staging/CONTRIBUTING.md#a-word-about-project-descriptions) and change your description accordingly so we can approve your PR

Reply:
gotcha will do